shell/server/pre-meteor.js

```diff
@@ -471,6 +490,12 @@ Meteor.startup(() => {
         return;
       }
 
+      if (id.match(/^selftest/)) {
```

Comment:
FWIW I think making hyphens be allowed is a one-char change to a regex in db.js.